src/ui/server/content-strings/pages/index.ts

```diff
@@ -40,6 +40,9 @@ const TALK_TO_AN_EXPORT_FINANCE_MANAGER_EXIT = {
       HREF: LINKS.EXTERNAL.EXPORT_FINANCE_MANAGERS,
     },
     TEXT: `to learn more about your options.`,
+    REASON: {
+      HIGH_COVER_HIGH_RISK_COUNTRY: "We do not normally offer short term cover for the country and cover perecentage you've selected.",
```

Comment:
```suggestion
      HIGH_RISK_COUNTRY_COVER_ABOVE_THRESHOLD: "We do not normally offer short term cover for the country and cover perecentage you've selected.",
```

I know it's longer but is a lot clearer